src/data/move.ts

```diff
@@ -1082,15 +1087,11 @@ export class PreMoveMessageAttr extends MoveAttr {
   }
 }
 
-export class StatusMoveTypeImmunityAttr extends MoveAttr {
-  public immuneType: Type;
-
-  constructor(immuneType: Type) {
-    super(false);
-
-    this.immuneType = immuneType;
-  }
-}
+/**
+ * Attribute for Status moves that take attack type effectiveness
+ * into consideration (i.e. {@linkcode https://bulbapedia.bulbagarden.net/wiki/Thunder_Wave_(move) | Thunder Wave})
+ */
```

Comment:
```suggestion
 * into consideration (i.e. {@linkcode https://bulbapedia.bulbagarden.net/wiki/Thunder_Wave_(move) | Thunder Wave})
 * @extends MoveAttr
 */
```
NIT: Minor additional doc line